src/pages/IDO/index.tsx

```diff
@@ -0,0 +1,336 @@
+          {showUnlockVoting ? (
+            <ButtonPrimary
+              style={{ width: 'fit-content' }}
+              padding="8px"
+              borderRadius="8px"
+              onClick={toggleDelegateModal}
+            >
+              {t('votePage.unlockVoting')}
+            </ButtonPrimary>
+          ) : availableVotes && JSBI.notEqual(JSBI.BigInt(0), availableVotes?.raw) ? (
+            <TYPE.body fontWeight={500} mr="6px">
+              <FormattedCurrencyAmount currencyAmount={availableVotes} /> {t('votePage.votes')}
+            </TYPE.body>
+          ) : pngBalance &&
+            userDelegatee &&
+            userDelegatee !== ZERO_ADDRESS &&
+            JSBI.notEqual(JSBI.BigInt(0), pngBalance?.raw) ? (
+            <TYPE.body fontWeight={500} mr="6px">
+              <FormattedCurrencyAmount currencyAmount={pngBalance} /> {t('votePage.votes')}
+            </TYPE.body>
+          ) : (
+            ''
+          )} */}
+        </WrapSmall>
+        {/* I don't think we need this section
+        {!showUnlockVoting && (
+          <RowBetween>
+            <div />
+            {userDelegatee && userDelegatee !== ZERO_ADDRESS ? (
+              <RowFixed>
+                <TYPE.body fontWeight={500} mr="4px">
+                  {t('votePage.delegatedTo')}
+                </TYPE.body>
+                <AddressButton>
+                  <StyledExternalLink
+                    href={getEtherscanLink(ChainId.FUJI, userDelegatee, 'address')}
+                    style={{ margin: '0 4px' }}
+                  >
+                    {userDelegatee === account ? 'Self' : shortenAddress(userDelegatee)}
+                  </StyledExternalLink>
+                  <TextButton onClick={toggleDelegateModal} style={{ marginLeft: '4px' }}>
+                    ({t('votePage.edit')})
+                  </TextButton>
+                </AddressButton>
+              </RowFixed>
+            ) : (
+              ''
+            )}
+          </RowBetween>
+        )}
+        */}
+
+        {/* This is the piece displayed when no IDOs were found 
+        {allProposals?.length === 0 && (
+          <EmptyProposals>
+            <TYPE.body style={{ marginBottom: '8px' }}>{t('IDOPage.noIDOsFound')}</TYPE.body>
+            <TYPE.subHeader>
+              <i>{t('IDOPage.IDOAnnounced')}</i>
+            </TYPE.subHeader>
+          </EmptyProposals>
+        )} */}
+        
+        {/* This is the primary IDO list. Here, it seems pages are built dynamically*/}
+        {/*
+        {allProposals?.map((p: ProposalData, i) => {
+          return (
+            <Proposal as={Link} to={'/vote/' + p.id} key={i}>
+              <ProposalNumber>{p.id}</ProposalNumber>
+              <ProposalTitle>{p.title}</ProposalTitle>
+              <ProposalStatus status={p.status}>{p.status}</ProposalStatus>
+            </Proposal>
+          )
+        })}
+        */}
+        {
+          <Proposal as={Link} to={'IDO/1'} key={1}>
```

Comment:
link is not working. please verify

Reply:
I changed this to use an external link to the announcement of the IDO instead of handling details on pangolin. Please review